src/components/VendorSalesPage/VendorSalesPage.js

```diff
@@ -162,6 +173,49 @@ class VendorSalesPage extends Component {
                 <Input addonBefore={prefixSelector} style={{ width: '100%' }} placeholder="Enter valid phone number" />
               )}
             </Form.Item>
+            <Form.Item 
+              label="Establishment Year" 
+            >
+              {getFieldDecorator('establishedYear', {
+                rules: [{ required: true, message: 'Please select an year!' }],
+              })(
+                <Select placeholder="Select an Year">
+                  {Years}
+                </Select>
+              )}
+            </Form.Item>
+            <Form.Item label="E-mail">
+          {getFieldDecorator('email', {
+            rules: [
+              {
+                type: 'email',
+                message: 'The input is not valid E-mail!',
+              },
+              {
+                required: false
+              },
+            ],
+          })(<Input />)}
+          </Form.Item>
+          <Form.Item
+              label="Register your website"
+            >
+              {getFieldDecorator('website', {
+                rules: [{ required: false, whitespace: true }],
+              })(
+                <Input placeholder="Enter Website" />
+              )}
+          </Form.Item>
+          <Form.Item label="Opening Time">
+          {getFieldDecorator('openingTime',{
+            rules: [{required:true}],
+          })( <TimePicker use12Hours format="h:mm a"  />)}
```

Comment:
Currently the user can set closing time before opening time. Correct that.
Also the antd timepicker isn't that great. Try finding something which is more user friendly. 
You may use different library for it. Before implementing this please share the link of the library.